test/abstract_syntax_tree.coffee

```diff
@@ -2281,17 +2281,22 @@ test "AST as expected for Assign node", ->
       name: 'c'
       declaration: no
 
-  testExpression 'a ?= b',
```

Comment:
Shouldn’t we also have this test?

Reply:
This no longer compiles now that we added the comparable error handling during AST generation (`a ?= b` throws an error during compile-to-JS), so I just adjusted this test to be valid Coffeescript (by embedding this expression inside a bigger expression that makes it valid)